src/Superdupercprogram.cpp

```diff
@@ -0,0 +1,11 @@
+//
+// Created by sebas on 24-10-2023.
+//
+#include <iostream>
+#include "Superdupercprogram.h"
+
+
+int main() {
+    cout << "poop is equal to " << poop;
```

Comment:
Vi foretrækker printf... eller noget (manglede noget at review)
```suggestion
#include <stdio>
#include "Superdupercprogram.h"


int main() {
    printf("poop is equal to %d", poop);
```

Reply:
```suggestion
#include <iostream>

int main() {
    int poop = 2;
    cout << "poop is equal to " << poop;
```